CHANGELOG.md

```diff
@@ -1,3 +1,338 @@
+  cd -
+  ```
+
+7. Remove old asset files.
+
+  ```bash
+  rm -rf $ASSETS_DIR/cluster-assets
+  rm -rf $ASSETS_DIR/terraform-modules
+  ```
+
+8. Update control plane.
+
+  ```bash
+  lokoctl cluster apply --skip-components -v
+  ```
+
+  > **NOTE:** If the update process gets interrupted, rerun above command.
+
+  > **NOTE:** If you are running self-hosted kubelet then append the above command with flag `--upgrade-kubelets`.
+
+  The update process typically takes about 10 minutes.
+  After the update, running `lokoctl health` should result in an output similar to the following:
+
+  ```bash
+  Node                     Ready    Reason          Message
+
+  lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+  Name      Status    Message              Error
+
+  etcd-0    True      {"health":"true"}
+  ```
+
+9. Update the bootstrap components: kubelet and etcd.
+
+  This script shipped with release tarball will update all the nodes to run the latest kubelet and etcd.
+
+  > **NOTE**: Please edit this script to disable updating certain nodes. Modify `update_other_nodes` function as required.
+
+  ```bash
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh true
+  ```
+
+10. Reload the new config for self-hosted kubelet.
+
+  ```bash
+  kubectl -n kube-system rollout restart ds kubelet
+  ```
+
+  If the above command failed, probably you are not running self-hosted kubelet, which is fine because the script in step 3 already updated the host kubelet.
+
+#### Update docker log settings
+
+> **NOTE**: This is an optional step if a specific node has humongous-log-generating pods then run these steps on that node.
+
+- SSH into the node and become root with `sudo -s`.
+- Create the docker config file:
+
+  ```json
+  echo '
+  {
+    "live-restore": true,
+    "log-opts": {
+      "max-size": "100m",
+      "max-file": "3"
+    }
+  }
+  ' | tee /etc/docker/daemon.json
+  ```
+
+- Restart docker daemon:
+
+  > **NOTE**: This will restart all the containers on the node, including kubelet. This step cannot be part of the automatic update script because restarting docker daemon will also kill the update script pod.
+
+  ```bash
+  systemctl restart docker
+  ```
+
+#### Updating Contour
+
+Manually update the CRDs before updating the component `contour`:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/v0.6.0/assets/charts/components/contour/crds/01-crds.yaml
+```
+
+Update the component:
+
+```bash
+lokoctl component apply contour
+```
+
+#### Updating Velero
+
+Manually update the CRDs before updating the component `velero`:
+
+```bash
+kubectl apply -f ./lokomotive-0.6.0/assets/charts/components/velero/crds/
```

Comment:
Why not use the github URL?

If you want to use the local path I'd use `$ASSETS_DIR`.

Reply:
> Why not use the github URL?

There are many files in that dir to use the github URL.

> If you want to use the local path I'd use $ASSETS_DIR.

Assets directory does not have velero configuration.